--- cairo-dock-core-3.5.2/CMakeLists.txt-	2025-07-26 12:34:31.820249272 +0000
+++ cairo-dock-core-3.5.2/CMakeLists.txt	2025-07-26 12:35:37.372408183 +0000
@@ -67,19 +67,13 @@
 
 set (prefix ${CMAKE_INSTALL_PREFIX})  # /usr/local
 set (exec_prefix ${prefix})
-set (datadir "${prefix}/${CMAKE_INSTALL_DATAROOTDIR}")  # (...)/share
+set (datadir "${CMAKE_INSTALL_DATAROOTDIR}")  # (...)/share
 set (pkgdatadir "${datadir}/${CMAKE_PROJECT_NAME}")  # (...)/cairo-dock
-set (mandir "${prefix}/${CMAKE_INSTALL_MANDIR}")  # (...)/man
+set (mandir "${CMAKE_INSTALL_MANDIR}")  # (...)/man
 
-if( CMAKE_SIZEOF_VOID_P EQUAL 8 AND (force-lib64 OR "${FORCE_LIB64}" STREQUAL "yes"))  # 64bits and force install in lib64
-	set (libdir "${prefix}/lib64")
-elseif (NOT "${LIB_SUFFIX}" STREQUAL "")
-	set (libdir "${prefix}/lib${LIB_SUFFIX}")  # (...)/libXX ## some distro use ${LIB_SUFFIX}
-else()
-	set (libdir "${prefix}/${CMAKE_INSTALL_LIBDIR}")  # (...)/lib or (...)/lib64 or custom ## GNU Install dir
-endif()
+set (libdir "${CMAKE_INSTALL_LIBDIR}")  # (...)/lib or (...)/lib64 or custom ## GNU Install dir
 
-set (includedir "${prefix}/${CMAKE_INSTALL_INCLUDEDIR}")  # (...)/include
+set (includedir "${CMAKE_INSTALL_INCLUDEDIR}")  # (...)/include
 set (bindir "${prefix}/${CMAKE_INSTALL_BINDIR}")  # (...)/bin
 
 set (pluginsdir "${libdir}/cairo-dock")
@@ -201,7 +195,7 @@
 set (CAIRO_DOCK_SHARE_DATA_DIR ${pkgdatadir})
 set (CAIRO_DOCK_SHARE_THEMES_DIR ${pkgdatadir}/themes)
 #set (CAIRO_DOCK_MODULES_DIR ${libdir}/cairo-dock)
-set (CAIRO_DOCK_LOCALE_DIR "${prefix}/${CMAKE_INSTALL_LOCALEDIR}")
+set (CAIRO_DOCK_LOCALE_DIR "${CMAKE_INSTALL_LOCALEDIR}")
 set (CAIRO_DOCK_THEMES_DIR "themes")  # folder name where themes are saved locally, relatively to the root folder of Cairo-Dock.
 set (CAIRO_DOCK_DISTANT_THEMES_DIR "themes3.4")  # folder name where themes are on the server, relatively to the root folder of the server files.
 set (CAIRO_DOCK_GETTEXT_PACKAGE ${GETTEXT_PACKAGE})
